comfy_extras/nodes_constant_values,.py

```diff
@@ -0,0 +1,79 @@
+class _CONSTANT_BASE:
```

Comment:
the name of this file has a comma in it?

Reply:
should be fixed now.